Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
The official Java repo on Docker Hub is : https://github.com/docker-library/openjdk and there is only openjdk

This repo https://github.com/dockerfile/java is not updated anymore, probably because licenses and trademarks

@jdubois : can we keep https://github.com/jhipster/generator-jhipster/blob/master/Dockerfile#L24 or should we use openjdk8 instead ?